group.hpp

```diff
@@ -78,7 +78,7 @@ class Group : public GroupInherit
 
     /** @brief Custom callback when LED group is asserted
      */
```

Comment:
Please add a small documentation about the callback function signature - explaining what the parameters and the return are.